### PR TITLE
Add documentation for disable body signing in S3

### PR DESCRIPTION
**Pull request by cjyclaire**
This PR update FAQ in SDK guide for disabling body signing in S3 as an option.

cc: @xibz @petermoon @mtdowling @jeskew

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -212,3 +212,31 @@ recommended that you disable content decoding at the S3 client level.
...
+
+    $s3Client = new Aws\S3\S3Client([
+        'version' => '2006-03-01',
+        'region'  => 'us-standard'
+    ]);
+
+    $params = [
+        'Bucket' => 'foo',
+        'Key'    => 'baz',
+        'ContentSha256' => 'UNSIGNED PAYLOAD'
```
Comment (xibz) [inline docs/faq.rst]: `UNSIGNED-PAYLOAD` for consistency with `s3.getPresignedPayload`

**Comment by xibz** [comment]
Should `UNSIGNED-PAYLOAD` be stored as a constant to drive consistency?

File: docs/faq.rst (hunk 1 above)
Comment (cjyclaire) [inline docs/faq.rst]: Make sense, I'll make that consistent.

**Comment by cjyclaire** [comment]
I can see the point in having it as a constant, yet, it seems now 'UNSIGNED-PAYLOAD' is defined in `getPresignedPayload()` only.

**Comment by jeskew** [comment]
If you define the value as a constant in `S3SignatureV4` (*not* `SignatureV4`), then you can refer to the constant in `getPresignedPayload` and in the documentation.

**Comment by cjyclaire** [comment]
Thanks to all.

**Comment by xibz** [comment]
Looks good. :shipit:

**Pull request merged**